plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -52,23 +52,19 @@ test.describe.serial( 'Add New Variable Product Page', () => {
 
 		// add 3 attributes
 		for ( let i = 0; i < 3; i++ ) {
-			if ( i > 0 ) {
-				await page.click( 'button.add_attribute' );
-			}
-
-			await page
-				.locator(
-					`.woocommerce_attribute_data input[name="attribute_names[${ i }]"]`
-				)
-				.fill( `attr #${ i + 1 }` );
-			await page
-				.locator(
-					`.woocommerce_attribute_data textarea[name="attribute_values[${ i }]"]`
-				)
-				.fill( 'val1 | val2' );
+			await page.click( 'button.add_attribute' );
+			await page.waitForSelector( `input[name="attribute_names[${ i }]"]` );
+			await page.fill(
+				`input[name="attribute_names[${ i }]"]`,
+				`attr #${ i + 1 }`
+			);
+			await page.fill(
+				`textarea[name="attribute_values[${ i }]"]`,
+				'val1 | val2'
+			);
 		}
-		await page.keyboard.press( 'ArrowUp' );
-		await page.click( 'text=Save attributes' );
+		await page.click( 'text=Save attributes', { force: true } );
+		await page.waitForLoadState( 'networkidle' );
```

Comment:
I also think we'd still need the `await page.keyboard.press( 'ArrowUp' );` action before clicking "Save attributes", otherwise this Save button will remain disabled 🤷‍♂️ for some reason.

_Maybe_ one of the input fields need to receive an actual human typing action before enabling the "Save attributes" button? If so, we can probably replace the `page.fill`'s above with `page.locator(...).type(...)` as mentioned [here](https://playwright.dev/docs/input#type-characters).